### PR TITLE
Add python38 build to one spec

### PR DESCRIPTION
**Pull request by mnecas**
Move the python38 SDK build to one spec file.

### CODE REVIEW

File: python-ovirt-engine-sdk4.spec.in [hunk 1]
```diff
@@ -27,16 +27,47 @@ Requires: python3-six
...
+This package contains the Python 3 SDK for version 4 of the oVirt Engine
+API.
+
+%files -n python38-ovirt-engine-sdk4
+%doc README.adoc
+%doc examples
+%license LICENSE.txt
+%define python3_pkgversion 38
+%define __python3 /usr/bin/python3.8
+%{python3_sitearch}/*
```
[the remark this answers is not in the document]
Comment (mnecas) [reply python-ovirt-engine-sdk4.spec.in:49]: The macro `python38_sitearch` does not exist so I added the definitions of `python3_pkgversion` and `__python3` so it overwrites the `python3_sitearch` path.

[the remark this answers is not in the document]
Comment (mwperina) [reply python-ovirt-engine-sdk4.spec.in:49]: OK

File: python-ovirt-engine-sdk4.spec.in [hunk 2]
```diff
@@ -27,16 +27,47 @@ Requires: python3-six
...
+Requires: libxml2
+Requires: python38
+Requires: python38-pycurl >= 7.43.0-6
+Requires: python38-six
+
+%description -n python38-ovirt-engine-sdk4
+This package contains the Python 3 SDK for version 4 of the oVirt Engine
+API.
+
+%files -n python38-ovirt-engine-sdk4
```
Comment (mwperina) [inline python-ovirt-engine-sdk4.spec.in:43]: To make the spec more readable wouldn't it be better to move 38 files section after normal files section?

File: python-ovirt-engine-sdk4.spec.in [hunk 3]
```diff
@@ -27,22 +27,51 @@ Requires: python3-six
...
+%package -n python38-ovirt-engine-sdk4
+Summary: oVirt Engine Software Development Kit (Python)
+BuildRequires: python38-devel
+Requires: libxml2
+Requires: python38
+Requires: python38-pycurl >= 7.43.0-6
+Requires: python38-six
+
+%description -n python38-ovirt-engine-sdk4
+This package contains the Python 3 SDK for version 4 of the oVirt Engine
```
Comment (mwperina) [inline python-ovirt-engine-sdk4.spec.in:40]: Python 3.8

File: python-ovirt-engine-sdk4.spec.in [hunk 4]
```diff
@@ -27,22 +27,51 @@ Requires: python3-six
...
+API.
+%endif
+
 %prep
 %setup -c -q -n ovirt-engine-sdk-python-@PACKAGE_VERSION@
 
-%build -n python3-ovirt-engine-sdk4
+%build
 %py3_build
```
Comment (mwperina) [inline python-ovirt-engine-sdk4.spec.in:48]: Hmm, do we need also py3_build or py38_build there?

**Review by mwperina — APPROVED** [review]
Looks good to me, but please wait for Marcin's review

**Review by tinez — APPROVED** [review]
Went over the built RPMs and didn't spot anything wrong. This is impressive really :)

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**